src/PowerShellEditorServices/Services/Workspace/WorkspaceService.cs

```diff
@@ -152,8 +152,19 @@ public ScriptFile GetFile(DocumentUri documentUri)
         /// </summary>
         /// <param name="filePath">The file path at which the script resides.</param>
         /// <param name="scriptFile">The out parameter that will contain the ScriptFile object.</param>
-        public bool TryGetFile(string filePath, out ScriptFile scriptFile) =>
-            TryGetFile(new Uri(filePath), out scriptFile);
+        public bool TryGetFile(string filePath, out ScriptFile scriptFile)
+        {
+            // This might not have been given a file path, in which case the Uri constructor barfs.
+            try
+            {
+                return TryGetFile(new Uri(filePath), out scriptFile);
+            }
+            catch (UriFormatException)
+            {
+                scriptFile = null;
+                return false;
+            }
+        }
```

Comment:
I had to change this one to catch the exception instead of using `Uri.IsWellFormedUriString(path, UriKind.RelativeOrAbsolute)` since the path we were being given was like `\\foo\\bar`, which makes me hesitant about the above too.